iris/tests/test_pipeline_health_checker.py

```
@@ -0,0 +1,146 @@
+    assert non_baseline == {"m_v2", "m_advanced"}
+
+
+class _FakeFeature(str, Enum):
+    FAKE = "FAKE"
+
+
+class _FakePipelineBase:
+    """Fake pipeline whose get_variants() is controlled by class attributes."""
+
+    HEALTH_BASELINE_VARIANT_ID = "default"
+    _VARIANTS: list[Variant] = []
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)